### DISCUSSION TITLE
Rabbitmq Management Http Apis

### DISCUSSION THREAD

**Discussion by mehdihadeli** [rabbitmq/rabbitmq-dotnet-client#1281] [Q&A] [answered]
Hi,
Is there any [managemt Http API](https://raw.githack.com/rabbitmq/rabbitmq-management/rabbitmq_v3_6_0/priv/www/api/index.html) client for RabbitMQ in c# like [this library](https://github.com/michaelklishin/rabbit-hole)?

**Comment by lukebakken** [comment, marked as answer]
https://www.google.com/search?q=rabbitmq+HTTP+API+client+dotnet

...finds the following:

* https://github.com/EasyNetQ/EasyNetQ.Management.Client
* https://github.com/thinkco/rabbitmq-httpclient-dotnet

**Reply by mehdihadeli** [reply]
Thanks :)
